CalibMuon/DTDigiSync/plugins/DTTTrigSyncFromDB.cc

```diff
@@ -103,24 +113,12 @@ class DTTTrigSyncFromDB : public DTTTrigBaseSync {
   std::string thet0Label;
 };
 
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/Framework/interface/ESHandle.h"
-#include "Geometry/DTGeometry/interface/DTLayer.h"
-#include "Geometry/DTGeometry/interface/DTSuperLayer.h"
-#include "DataFormats/MuonDetId/interface/DTWireId.h"
-#include "CondFormats/DTObjects/interface/DTT0.h"
-#include "CondFormats/DataRecord/interface/DTT0Rcd.h"
-#include "CondFormats/DTObjects/interface/DTTtrig.h"
-#include "CondFormats/DataRecord/interface/DTTtrigRcd.h"
-
-#include <iostream>
-
 using namespace std;
 using namespace edm;
 
-DTTTrigSyncFromDB::DTTTrigSyncFromDB(const ParameterSet& config)
-    : debug(config.getUntrackedParameter<bool>("debug")),
+DTTTrigSyncFromDB::DTTTrigSyncFromDB(const ParameterSet& config, edm::ConsumesCollector cc)
+    : ttrigToken_(cc.esConsumes()),
```

Comment:
I should have used theTrigLabel here

Reply:
I just checked all `.get(` statements in my change looking for any other missed uses of a label. I didn't find any.